components/client-attestation-mgt/org.wso2.carbon.identity.client.attestation.mgt/pom.xml

```diff
@@ -177,6 +177,15 @@
                     </instructions>
                 </configuration>
             </plugin>
+            <!--  Class AndroidAttestationServiceCredentials not following checkstyle format for parameter name
+                   due to Google Service Account JSON key format. Hence adding checkstyle skip-->
```

Comment:
Instead of changing the naming format cann't something like jackson annotation be used?

Reply:
@SerializedName can be used for gson and @JsonProperty can be used for jackson depending on the library used for parsing.